daemon/graphdriver/aufs/aufs.go

```diff
@@ -573,7 +573,7 @@ func (a *Driver) aufsMount(ro []string, rw, target, mountLabel string) (err erro
 
 	offset := 54
 	if useDirperm() {
-		offset += len("dirperm1")
+		offset += len(",dirperm1")
```

Comment:
FWIW, I was thinking something like

```go
	opts := []string{"dio","xino=/dev/shm/aufs.xino"}
	if useDirperm() {
		opts = append(opts, "dirperm1")
	}
	offset := len(strings.Join(opts,","))
```

To prevent the magic `54` number here. But I'm trying to find where exactly the 54 is based on

Looks like it was introduced in https://github.com/moby/moby/commit/6d97339ca23ada27812572016ad4ff9ccffa8b09#diff-8a0595bbcb3e2079a9f6a87bfccb1038R712, so perhaps @tonistiigi could help 😅 (also if it makes sense to change)